postgres/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 - The format is based on [Keep a Changelog](http://keepachangelog.com/).
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Version 1.3.2 - tbd
+
+### Changed
+
+- Using the new `cds build` plugin API (requires `@sap/cds-dk` 7.3.1)
```

Comment:
You dont have to take care of the changelog here, just follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) convention and the changelog will be automagically generated for you. :)